yt_dlp/extractor/youtube/_video.py

```diff
@@ -3348,7 +3385,107 @@ def process_https_formats():
 
                 yield from https_fmts
 
+            def process_sabr_formats_and_subtitles():
+                proto = 'sabr'
+                server_abr_streaming_url = streaming_data.get('serverAbrStreamingUrl')
+
+                query = parse_qs(server_abr_streaming_url)
+                if query.get('n'):
+                    n_challenge = query['n'][0]
```

Comment:
Will need to find a way to bulk process these with https formats n/sig to avoid additional call to a js runtime